src/nuxeo-cloud/start-developing.md

```diff
@@ -0,0 +1,135 @@
+---
+title: Start Developing an Application
+review:
+    comment: ''
+    date: '05-14-2020'
+    status: ok
+labels:
+    - nuxeo-cloud
+tree_item_index: 400
+toc: true
+---
+
+## Concept
+
+The Nuxeo Dev Sandbox is built using OpenShift. OpenShift is an open source container application platform, based on top of Docker containers and Kubernetes pods. It enables you to easily build and deploy applications during their development process.
+
+Nuxeo uses the Dev Sandbox, as part of the development process, to deploy and validate work before pushing to production cloud environments.
+
+The Nuxeo Sandbox is meant purely for Development. It does not support large volume data storage, data backups and it should never store production data.
+
+## Environments
+
+During the application development process, there is a need to test the latest work on test environments other than pre-production or production environments.
```

Comment:
change this section to 

During the application development process, there is a need to test new configurations and changes in a development environment rather than Pre-Production or Production Environments.

The standard Development Sandbox project provides customers with 2 sandbox environments that can be managed directly via the OpenShift developer admin:

- Development Environment (DEV): Initial environment for exploration of updates that is continuously updated with the latest changes.
- Test Environment (UAT): Secondary environment for validation of updates that may require additional manual/user test efforts.

The DEV & UAT Environments share a MongoDB and Elasticsearch server, but each has its own MongoDB database and ElasticSearch index.